README.md

```diff
@@ -204,7 +205,7 @@ and use that as a temp location for this data.
  <tr>
     <td><tt>jdbcdriver</tt></td>
     <td>No</td>
-    <td><tt>org.postgresql.Driver</tt></td>
+    <td><tt>com.amazon.redshift.jdbc4.Driver</tt></td>
```

Comment:
Here, I chose to use the JDBC 4.0 driver instead of 4.1 in order to retain Java 6 support.